packages/forma-36-react-components/src/components/Note/Note.css

```diff
@@ -58,3 +65,19 @@
   display: flex;
   margin-top: 1px;
 }
+
+.Note__dismiss {
+  background: transparent;
+  border: none;
+  cursor: pointer;
+  outline: none;
+  margin-top: 0;
+  margin-right: 0;
+  margin-left: auto;
+  pointer-events: all;
+  display: flex;
+}
+
+.Note__dismiss span {
```

Comment:
If you take @domarku's suggestion for using `margin-bottom: auto` we can get rid of this bit of CSS completely